component/azstorage/utils.go

```diff
@@ -474,6 +474,25 @@ func getAccessTierType(name string) azblob.AccessTierType {
 	return azblob.AccessTierNone
 }
 
+// Called by x method
+func getACLPermissions(mode os.FileMode) string {
+	// Format for ACL and Permission string is different
+	// ACL:"user::rwx,user:<id>:rwx,group::rwx,mask::rwx,other::rwx"
+	// Permissions:"rwxrwxrwx+"
+	// If we call the set ACL without giving user then all other principals will be removed.
```

Comment:
"If we call the set ACL without giving user then all other principals will be removed." could you explain what this means

Reply:
From portal if you do "Manage ACL" to any file/folder in container there are two things in it. one is regular linux style triplet of 'rwx' and additionally you can provide specific access to various principals and identities as well. If we call SetACL it just overwrites all of them because we pass only the 'rwx' triplet and not the other principal related info. While we set permissions it just overwrites the triplets and leave the principals as is.